soql-stdlib/src/main/scala/com/socrata/soql/functions/SoQLFunctions.scala

```diff
@@ -453,6 +453,9 @@ object SoQLFunctions {
   val FloatingTimeStampExtractWoy = mf("floating timestamp extract week of year", FunctionName("date_extract_woy"), Seq(SoQLFloatingTimestamp), Seq.empty, SoQLNumber)(
     NoDocs
   )
+  val FloatingTimestampExtractIsoY = mf("floating timestamp extract isoyear", FunctionName("date_extract_iso_y"), Seq(SoQLFloatingTimestamp), Seq.empty, SoQLNumber)(
+    NoDocs
```

Comment:
`NoDocs` :(

If you write a doc, it'll show up in the new query editor :)

Reply:
Where should I write said doc? Is there an example?

Reply:
The discussion happened elsewhere. There is another branch which is doing a bunch of the initial doc write-ups (https://github.com/socrata-platform/soql-reference/tree/cduranti/add-docs). Once that branch is merged, I'll loop back around making a new pr to add the doc for this new function.